packages/gatsby/src/schema/types/__tests__/date.js

```diff
@@ -0,0 +1,453 @@
+const { store } = require(`../../../redux`)
+const { build } = require(`../..`)
+const { isDate } = require(`../date`)
+
+// Timestamps grabbed from https://github.com/moment/moment/blob/2e2a5b35439665d4b0200143d808a7c26d6cd30f/src/test/moment/is_valid.js
+
+describe(`isDate`, () => {
+  beforeEach(async () => {
+    // pass
+  })
+
+  it(`supports valid ISO 8601 datetimes`, async () => {
```

Comment:
maybe refactoring these into `it.each` style would be beneficial for test run purposes, since when using `.each` all are run, even if any fail, rather than bailing early

ie.

```js
it.each([
  '1970',
  '2019',
  '1970-01',
  ...
])('should return true for %s', dateString => {
  expect(isDate(dateString)).toBeTruthy();
})
```

Reply:
Thanks @harryparkdotio 	for the advice, went ahead and modified the tests according to what you mentioned.